tedge/src/cli/connect/command.rs

```diff
@@ -404,7 +404,13 @@ fn new_bridge(
     config_location: &TEdgeConfigLocation,
 ) -> Result<(), ConnectError> {
     println!("Checking if {} is available.\n", service_manager.name());
-    let () = service_manager.check_operational()?;
+    let service_manager_result = service_manager.check_operational();
+
+    if let Err(SystemServiceError::ServiceManagerUnavailable(name)) = &service_manager_result {
+        println!("It appears that '{}' service manager is installed on the system, thin-edge.io works seamlessly with 'systemd'.", name);
+        println!("'tedge connect' will create MQTT configuration, but you will have to start required services on your own.");
+        println!("Start following services: mosquitto, tedge_mapper, tedge_agent.");
```

Comment:
```suggestion
        println!("Start/Restart the following services: mosquitto, tedge-mapper-*, tedge-agent.");
```

Reply:
It's not as straightforward herr with tedge_mapper.
What you are suggesting with the * would most likely not apply to no systemd installation as services are specific to systemd.
The user would have to execute the tedge_mapper binary manually and therefore they will get hints about available mappers from hints.

Reply:
I guess the safer option is to say "Restart mosquitto and start other thin edge components"